src/GregClientSandbox/GregClientSandbox.csproj

```diff
@@ -42,7 +42,7 @@
       <HintPath>..\..\..\..\..\..\Desktop\DotNetOpenAuth-4.2.2.13055\Lib\log4net.dll</HintPath>
```

Comment:
I'm not sure why is this platform of this project being set to `x86`, do you want me to update it to `AnyCPU`?